doc/source/Tutorials/BeginnerTutorial/index.md

```diff
@@ -138,39 +140,56 @@ Before you can start any rendering, you will need to create a render camera. Jus
 
 Earlier, you learned how to find a node by navigating the node creation menu in the _Graph Editor_. If you know the name of the node you wish to create, and do not feel like navigating the menu, you can instead use the menu's search feature:
 
-1. Hover the cursor over the background in the _Graph Editor_.
+1. Right-click the background of the _Graph Editor_. The node creation menu will open.
 
-2. Right-click. The node creation menu will open.
+    > Tip :
+    > With the cursor hovering over the _Graph Editor_, you can also hit <kbd>Tab</kbd> to open the node creation menu.
 
-> Tip :
-> With the cursor hovering over the _Graph Editor_, you can also hit <kbd>Tab</kbd> to open the node creation menu.
-
-3. Type `Camera`. A list of search results will appear. By default, _Camera_ will be highlighted.
+2. Type `Camera`. A list of search results will appear. By default, _Camera_ will be highlighted.
 
-4. Hit <kbd>Enter</kbd> to create a new Camera node.
+3. Hit <kbd>Enter</kbd> to select _Camera_. A new Camera node will appear in the _Graph Editor_.
 
     ![A new Camera node](images/mainCameraNode.png "A new Camera node")
 
-As before, the newly created node will be selected automatically, and the _Viewer_, _Scene Hierarchy_, and _Node Editor_ will update to reflect this new selection.
+As before, the newly created node will be selected automatically, and the _Viewer_, _Hierarchy View_, and _Node Editor_ will update to reflect this new selection.
 
 
-## Nodes, Scenes, and Plugs ##
+## Node Data Flow ##
```

Comment:
I expanded this section based on an updated explanation of how the nodes are processed.

I couldn't find a reasonable way of signposting this section, but this is probably because this section should really be moved to its own article, and only briefly touched on here. It's certainly valuable info, but I don't think it belongs in this tutorial with this level of depth.